api/profile_test.go

```diff
@@ -12,12 +12,20 @@ import (
 	"testing"
 
 	"github.com/beme/abide"
+	"github.com/qri-io/qri/config"
+	"github.com/qri-io/qri/lib"
 )
 
 func TestProfileHandler(t *testing.T) {
 	node, teardown := newTestNode(t)
 	defer teardown()
 
+	cfg := config.DefaultConfigForTesting()
```

Comment:
Create a helper method that does this, and use it in these functions in order to call out the existence of the pattern:

```
  inst := newTestInstanceWithProfileFromNode(node)
```

```
func newTestInstanceWithProfileFromNode(node *p2p.QriNode) *instance {
  cfg := config.DefaultConfigForTesting()
  pro, _ := node.Repo.Profile()
  cfg.Profile, _ = pro.Encode()
  return &instance{cfg: cfg, node: node}
}
```